### PR TITLE
fix kube-push for ubuntu

### PR DESCRIPTION
**Pull request by dalanlan**
/cc @mikedanese @WIZARD-CXY

### CODE REVIEW

**Comment by mikedanese** [comment]
Thanks for doing this cleanup! @dalanlan I will review today.

File: cluster/ubuntu/build.sh [hunk 1]
```diff
@@ -54,10 +53,10 @@ cp $ETCD/etcd $ETCD/etcdctl binaries/master
 
 # k8s
 echo "Download kubernetes release ..."
-K8S_VERSION=${K8S_VERSION:-"1.0.3"}
+KUBE_VERSION=${KUBE_VERSION:-"1.0.3"}
```
Comment (WIZARD-CXY) [inline cluster/ubuntu/build.sh]: why change to the KUBE_VERSION??

**Comment by dalanlan** [comment]
> Does #13381 break anything? 

Yes, i'm afraid.
- If you use `./kube-push.sh -n <master-ip> <kube-version>`, it would leave your cluster alive w/o a master by replacing it with a working node. Or i wont even call it *alive*, maybe.
- The old one would never push the required release at all, 'cause it never propagate "$KUBE_VERSION" var to prepare-push
- Incorrect logic of roles judgement within *for* loop, also for the prepare-push, which makes all of the binaries unavailable for kube-push
- ...

File: cluster/ubuntu/build.sh (hunk 1 above)
Comment (dalanlan) [inline cluster/ubuntu/build.sh]: To propagate $KUBE_VERSION var which indicates a required release
https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-push.sh#L70

Comment (dalanlan) [inline cluster/ubuntu/build.sh]: Or we can use `K8S_VERSION=$KUBE_VERSION` in prepare-push func, which seems a little messy to me.

**Comment by dalanlan** [comment]
I cannot really access details for Shippable (shame). Try closing and re-opening to see if it's possible to trigger it.

**Comment by mikedanese** [comment]
LGTM thanks!

**Pull request merged**